src/containers/CreateArticle/TextEditor.js

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from "react";
+import { Editor } from "@tinymce/tinymce-react";
+import PropTypes from "prop-types";
+
+const apiKey = "0naso5vy7fw5eu2eqblor00o1c326x0hupermg4cli0rqoqn";
```

Comment:
This should be stored in the `.env` file

Reply:
I have done that.